nimble/host/audio/services/bass/include/services/bass/ble_audio_svc_bass.h

```diff
@@ -0,0 +1,450 @@
+
+/** Value of not assigned Source ID of Receiver State */
+#define BLE_SVC_AUDIO_BASS_RECEIVE_STATE_SRC_ID_NONE                     0xFF
+
+/** BLE AUDIO BASS Add Source operation OP code */
+#define BLE_SVC_AUDIO_BASS_OPERATION_ADD_SOURCE                          0x01
+
+/** BLE AUDIO BASS Modify Source operation OP code */
+#define BLE_SVC_AUDIO_BASS_OPERATION_MODIFY_SOURCE                       0x02
+
+/** BLE AUDIO BASS Remove Source operation OP code */
+#define BLE_SVC_AUDIO_BASS_OPERATION_REMOVE_SOURCE                       0x03
+
+/** BLE AUDIO BASS Error: OP Code not supported */
+#define BLE_SVC_AUDIO_BASS_ERR_OPCODE_NOT_SUPPORTED                      0x80
+
+/** BLE AUDIO BASS Error: Invalid Source ID */
+#define BLE_SVC_AUDIO_BASS_ERR_INVALID_SOURCE_ID                         0x81
+
+/** BLE AUDIO BASS Encryption States */
+enum ble_svc_audio_bass_big_enc {
+    /** BLE AUDIO BASS BIG Encryption: Not Encrypted */
+    BLE_SVC_AUDIO_BASS_BIG_ENC_NOT_ENCRYPTED,
+
+    /** BLE AUDIO BASS BIG Encryption: Broadcast Code Required */
+    BLE_SVC_AUDIO_BASS_BIG_ENC_BROADCAST_CODE_REQ,
+
+    /** BLE AUDIO BASS BIG Encryption: Decrypting */
+    BLE_SVC_AUDIO_BASS_BIG_ENC_DECRYPTING,
+
+    /** BLE AUDIO BASS BIG Encryption: Bad Code */
+    BLE_SVC_AUDIO_BASS_BIG_ENC_BAD_CODE
+};
+
+/** BLE AUDIO BASS PA Sync parameters, valid fo Modify Source operation */
+enum ble_svc_audio_bass_pa_sync {
+    /** BLE AUDIO BASS PA Sync: Do not synchronize to PA */
+    BLE_SVC_AUDIO_BASS_PA_SYNC_DO_NOT_SYNC,
+
+    /** BLE AUDIO BASS PA Sync: Synchronize to PA – PAST available */
+    BLE_SVC_AUDIO_BASS_PA_SYNC_SYNC_PAST_AVAILABLE,
+
+    /** BLE AUDIO BASS PA Sync: Synchronize to PA – PAST not available */
+    BLE_SVC_AUDIO_BASS_PA_SYNC_SYNC_PAST_NOT_AVAILABLE,
+
+    /**
+     * BLE AUDIO BASS PA Sync: reserved for future use.
+     * This shall be always last value in this enum
+     */
+    BLE_SVC_AUDIO_BASS_PA_SYNC_RFU
+};
+
+/** BLE AUDIO BASS Broadcast Receiver: PA Sync States */
+enum ble_svc_audio_bass_pa_sync_state {
+    /** BLE AUDIO BASS PA Sync State: Not synchronized to PA */
+    BLE_SVC_AUDIO_BASS_PA_SYNC_STATE_NOT_SYNCED,
+
+    /** BLE AUDIO BASS PA Sync State: SyncInfo Request */
+    BLE_SVC_AUDIO_BASS_PA_SYNC_STATE_SYNC_INFO_REQ,
+
+    /** BLE AUDIO BASS PA Sync State: Synchronized to PA */
+    BLE_SVC_AUDIO_BASS_PA_SYNC_STATE_SYNCED,
+
+    /** BLE AUDIO BASS PA Sync State: Failed to synchronize to PAA */
+    BLE_SVC_AUDIO_BASS_PA_SYNC_STATE_SYNCED_FAILED,
+
+    /** BLE AUDIO BASS PA Sync State: No PAST */
+    BLE_SVC_AUDIO_BASS_PA_SYNC_STATE_NO_PAST
+};
+
+/** BLE AUDIO BASS Broadcast Receiver State: Subgroup entry */
+struct ble_svc_audio_bass_subgroup {
+    /** BLE AUDIO BASS Subgroup entry: Bis Synchronization State */
+    uint32_t bis_sync_state;
+
+    /** BLE AUDIO BASS Subgroup entry: Metadata length */
+    uint8_t metadata_length;
+
+    /** BLE AUDIO BASS Subgroup entry: Metadata */
+    uint8_t *metadata;
```

Comment:
const?

Reply:
this pointer is taken from mempool - cant be const